packages/dx-grid-core/src/plugins/table-row-detail/constants.js

```diff
@@ -1 +1 @@
-export const TABLE_DETAIL_TYPE = 'detail';
+export const TABLE_DETAIL_TYPE = Symbol('detail');
```

Comment:
Why we don't use `.toString()` here?

Reply:
We use toString only in the key definition. Type should be symbol.